linera-chain/src/manager.rs

```diff
@@ -606,33 +602,47 @@ where
 
     /// Returns whether the signer is a valid owner and allowed to propose a block in the
     /// proposal's round.
-    pub fn verify_owner(&self, proposal: &BlockProposal) -> bool {
-        let owner = &proposal.public_key.into();
-        if self.ownership.get().super_owners.contains(owner) {
-            return true;
+    pub fn verify_owner(
+        &self,
+        proposal_owner: &AccountOwner,
+        proposal_round: Round,
+    ) -> Result<bool, CryptoError> {
+        if self.ownership.get().super_owners.contains(proposal_owner) {
+            return Ok(true);
         }
-        match proposal.content.round {
+        tracing::info!(
+            "Chain's super owners: {:?}, proposal_owner: {:?}",
+            self.ownership.get().super_owners,
+            proposal_owner
+        );
+        tracing::info!(
+            "Chain's owners: {:?}, proposal_owner: {:?}",
+            self.ownership.get().owners,
+            proposal_owner
+        );
```

Comment:
```suggestion
```

(Or make it `debug` or `tracing`, and just print the `ownership`; but I'd just remove them.)

Reply:
Definitely, this shouldn't get in.